services/documentation_service.py

```diff
@@ -214,34 +218,12 @@ async def _generate_doc_for_file(
             raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,
                                 detail="File cannot be empty")
 
-        # currently user_prompt is used for both the JSON and Markdown calls
-        user_prompt = f"Document the following code file titled {file.path}\n\n{file.decoded_content}"
```

Comment:
there were two functions with almost identical code. So I made a helper for them. That's why I'm deleting most of this code